client/src/document/ingredients/browser-compatibility-table/headers.tsx

```diff
@@ -13,7 +13,7 @@ export const PLATFORM_BROWSERS: { [key: string]: bcd.BrowserNames[] } = {
     "samsunginternet_android",
   ],
   server: ["nodejs"],
-  "webextensions-desktop": ["chrome", "edge", "firefox", "opera"],
+  "webextensions-desktop": ["chrome", "edge", "firefox", "opera", "safari"],
```

Comment:
Visit http://localhost:3000/en-US/docs/Mozilla/Add-ons/WebExtensions/API/dns and enjoy the glorious presence of the "Safari" column.